alvr/client_core/src/lib.rs

```diff
@@ -89,6 +89,11 @@ pub struct ClientCoreContext {
 
 impl ClientCoreContext {
     pub fn new(capabilities: ClientCapabilities) -> Self {
+        #[cfg(any(target_os = "ios", target_os = "macos"))]
+        {
+            init_logging();
+        }
```

Comment:
To initialize logging you need to use the C API. I realized that the function is missing, I added it in here #2439

Reply:
Ok, #2439 got merged, you just need to rebase.